24.md

```diff
@@ -0,0 +1,34 @@
+# NIP-24
+## Topics
+
+This NIP defines `kind:31000` (a parameterized replaceable event according to [NIP-33](https://github.com/nostr-protocol/nips/blob/master/33.md)) for topics that group a selection of articles of a `kind:30023` of the NIP-23.
+
+## Format
+The `.content` property should hold a `String` of a `JSON` syntax, a `Stringyfied` JSON.
+
+The detailed structure of the `Strigyfied` content should hold the following properties:
+- `title`
+- `excerpt`
+- `thumbnail`
+
+## Editability
+These articles are meant to be editable, so they should make use of the replaceability feature of NIP-33 and include a `d`  tag with an identifier for the article. Clients should take care to only publish and read these events from relays that implement that. If they don't do that they should also take care to hide old versions of the same article they may receive.
+
```

Comment:
This paragraph is not adding anything substantial. nip-33 already states that clients should not assume relays to support replacement and therefore take the last edit as **the** version. Even if the relay supports it, the client **will always** receive multiple versions whenever the article is being edited.

The fact that this event is meant to be replaceable is already stated in the very first sentence.

```suggestion
```